tests/integ/scala/test_datatype_suite.py

```diff
@@ -472,6 +474,118 @@ def test_structured_dtypes_iceberg(
         structured_type_session.sql(f"drop table if exists {table_name}")
 
 
+@pytest.mark.skip(
+    reason="SNOW-1748140: Need to handle structured types in datatype_mapper"
+)
+def test_struct_dtype_iceberg_lqb(
```

Comment:
Does this take into account the scenario we discussed yesterday or not yet?

Reply:
not yet. let me update the test